extensions/extension.go

```diff
@@ -324,6 +324,8 @@ func createJSONAndEvalFunctions(c *Config) {
 	}
 }
 
+const DefaultTrimSet = " \r\n\t"
```

Comment:
```suggestion
const defaultTrimSet = " \r\n\t"
```

Unless you plan to reuse it

Reply:
I figured it would be good to get it documented (thus exposed)